utilities/arrhelp/arrhelp.go

```diff
@@ -0,0 +1,14 @@
+package arrhelp
+
+/*
+IntIndexOf finds the index of target element
+it will return -1 while find nothing
+*/
+func IntIndexOf(elements []int, targetElement int) (index int) {
+	for index, element := range elements {
+		if element == targetElement {
+			return index
+		}
+	}
+	return -1
+}
```

Comment:
Would it be better to change the package directory to utils/array-utils?
There are no rules for this, but I think it is more suitable to name it as a utility function, instead of a helper function
[Reference](https://stackoverflow.com/questions/12192050/what-are-the-differences-between-helper-and-utility-classes)

Reply:
@jamfly

Reply:
@yaoandy107 done.